src/Sfa.Das.ApprenticeshipInfoService.Api/Controllers/AssessmentOrgsController.cs

```diff
@@ -133,9 +135,54 @@ public IEnumerable<Organisation> GetByStandardId(string id)
             return response;
         }
 
+        /// <summary>
+        /// Get standards by assessment organisation
+        /// </summary>
+        /// <returns>colllection of standards by specific organisation identifier</returns>
+        [SwaggerOperation("GetStandardsByOrganisationId")]
+        [SwaggerResponse(HttpStatusCode.OK, "OK", typeof(IEnumerable<StandardOrganisationSummary>))]
+        [Route("assessment-organisations/{organisationId}/standards", Name = "GetStandardsByOrganisationId")]
+        [ExceptionHandling]
+        public IEnumerable<StandardOrganisationSummary> GetStandardsByOrganisationId(string organisationId)
+        {
+            try
+            {
+                var response = _getAssessmentOrgs.GetStandardsByOrganisationIdentifier(organisationId).ToList();
```

Comment:
`if (_getStandards.GetOrganisationById(id) == null)
{
throw HttpResponseFactory.RaiseException(
HttpStatusCode.NotFound,
$"No organisation with EpaOrganisationIdentifier {id} found");
}`

can we add this logic?.